### PR TITLE
feat: Add the hasBackdrop property

### PR DESCRIPTION
**Pull request by emilianodevborn**
Problem

In the case of size=invisible the SDK still shows the modal backDrop

Solution

Add a new hasBackdrop property to handle the modal backDrop behaviour

### CODE REVIEW

**Comment by e271828-** [comment]
@emilianodevborn thanks! mind updating the README?

File: index.js [hunk 1]
```diff
@@ -55,7 +56,7 @@ class ConfirmHcaptcha extends PureComponent {
         onBackdropPress={() => this.hide('backdrop')}
         onBackButtonPress={() => this.hide('back_button')}
         isVisible={show}
-        hasBackdrop={!passiveSiteKey}
+        hasBackdrop={hasBackdrop}
```
Comment (zoryana94) [inline index.js:59]: Not to change the current behavior, please try out the following:
```
hasBackdrop={!passiveSiteKey && hasBackdrop}
```

**Review by zoryana94 — COMMENTED** [review]
Thank you for the contribution, @emilianodevborn! 👍

File: index.js (hunk 1 above)
Comment (emilianodevborn) [reply index.js]: Done! and updated the README

File: README.md [hunk 2]
```diff
@@ -127,6 +127,7 @@ Otherwise, you should pass in the preferred device locale, e.g. fetched from `ge
 | style _(inline component only)_ | ViewStyle (see [here](https://reactnative.dev/docs/view-style-props)) | The webview style |
 | baseUrl _(modal component only)_ | string | The url domain defined on your hCaptcha. You generally will not need to change this. |
 | passiveSiteKey _(modal component only)_ | boolean | Indicates whether the passive mode is enabled; when true, the modal won't be shown at all |
+| hasBackdrop _(modal component only)_ | boolean | To hide the modal backDrop; when true, the backDrop modal won’t be displayed |
```
Comment (zoryana94) [inline README.md:130]: it's hidden when the `hasBackdrop` is set to false, not vice versa

how about `Defines if the modal backdrop is shown (true by default)`?

Comment (emilianodevborn) [reply README.md]: Yes, you are right, there I made the changes

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hcaptcha/react-native-hcaptcha",
-  "version": "1.5.1",
+  "version": "1.5.2",
```
Comment (zoryana94) [inline package.json:3]: @emilianodevborn, please also update the version to 1.6.0 as it's technically the new feature

Comment (emilianodevborn) [reply package.json]: @zoryana94 Done!

**Review by e271828- — APPROVED** [review]

**Comment by vasylnahuliak** [comment]
Missing typescript support

```
Property 'hasBackdrop' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<ConfirmHcaptcha> & Readonly<ConfirmHcaptchaProps>'.
```

**Pull request merged**